packages/composites/curve-3pool/test/integration/fixtures.ts

```diff
@@ -0,0 +1,254 @@
+import nock from 'nock'
+
+export function mockEthereumCalls() {
+  nock('https://geth-main.eth.devnet.tools:443', { encodedQueryParams: true })
+    .persist()
+    .post('/', {
+      method: 'eth_chainId',
+      params: [],
+      id: /^\d+$/,
+      jsonrpc: '2.0',
+    })
+    .reply(200, (_, request) => ({ jsonrpc: '2.0', id: request['id'], result: '0x1' }), [])
+    .post('/', {
+      method: 'eth_call',
+      params: [{ to: '0x6c3f90f043a72fa612cbac8115ee7e52bde6e490', data: '0x18160ddd' }, 'latest'],
+      id: /^\d+$/,
+      jsonrpc: '2.0',
+    })
+    .reply(
+      200,
+      (_, request) => ({
+        jsonrpc: '2.0',
+        id: request['id'],
+        result: '0x0000000000000000000000000000000000000000054051140f7c4b26349ad44b',
+      }),
+      [],
+    )
+    .post('/', {
+      method: 'eth_call',
+      params: [
+        {
+          to: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+          data: '0x4903b0d10000000000000000000000000000000000000000000000000000000000000000',
+        },
+        'latest',
+      ],
+      id: /^\d+$/,
+      jsonrpc: '2.0',
+    })
+    .reply(
+      200,
+      (_, request) => ({
+        jsonrpc: '2.0',
+        id: request['id'],
+        result: '0x0000000000000000000000000000000000000000011a882458c863bb8aa6e218',
+      }),
+      [],
+    )
+    .post('/', {
+      method: 'eth_call',
+      params: [
+        {
+          to: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+          data: '0x4903b0d10000000000000000000000000000000000000000000000000000000000000001',
+        },
+        'latest',
+      ],
+      id: /^\d+$/,
+      jsonrpc: '2.0',
+    })
+    .reply(
+      200,
+      (_, request) => ({
+        jsonrpc: '2.0',
+        id: request['id'],
+        result: '0x000000000000000000000000000000000000000000000000000132a728e0d17e',
+      }),
+      [],
+    )
+    .post('/', {
+      method: 'eth_call',
+      params: [
+        {
+          to: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7',
+          data: '0x4903b0d10000000000000000000000000000000000000000000000000000000000000002',
+        },
+        'latest',
+      ],
+      id: /^\d+$/,
+      jsonrpc: '2.0',
+    })
+    .reply(
+      200,
+      (_, request) => ({
+        jsonrpc: '2.0',
+        id: request['id'],
+        result: '0x00000000000000000000000000000000000000000000000000037ca0681402ee',
+      }),
+      [],
+    )
+}
+
+export function mockTiingoResponse() {
+  nock('http://localhost:3000')
+    .post('/', { id: '1', data: { base: 'USDC', quote: 'USD', endpoint: 'price' } })
+    .reply(200, {
```

Comment:
If you're returning the same value every time for the same inputs, you can use `.persist()` and remove the other copies of the post/reply.

Reply:
Tbh, not familiar with nock, but I have two different interceptors for each post actually (just $1 for the first set and slightly off $1 for the next), so I think this is fine?

Reply:
Oh I see, yeah that's fine then